.travis.yml

```diff
@@ -1,7 +1,22 @@
+sudo: false
 language: go
 go:
- - 1.4
- - 1.5
- - tip
-install: go get -v ./github
-script: go test -v ./github
+  - 1.5.4
+  - 1.6.2
+  - tip
+matrix:
+  include:
+    - go: 1.4.3
+      script:
+        - go get -t -v ./...
+        - go test -v -race ./...
+  allow_failures:
+    - go: tip
+  fast_finish: true
+install:
+  - # Do nothing. This is needed to prevent default install action "go get -t -v ./..." from happening here (we want it to happen inside script step).
+script:
+  - go get -t -v ./...
+  - diff -u <(echo -n) <(gofmt -d -s .)
+  - go tool vet .
+  - go test -v -race ./...
```

Comment:
it looks like two files in tests/integration are missing the "integration" build constraint, so expanding this call to "./..." picks those up.  Could you add that build constraint to `tests/integration/authorizations_test.go` and `tests/integration/github_test.go` as part of this?